src/card.css

```diff
@@ -0,0 +1,114 @@
+.card {
+  background: var(--clear_white-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card--position {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  max-width: 200px;
+}
+
```

Comment:
merge them and remove modifier, it's useless

Reply:
fixed